lib/mixins/property-effects.html

```diff
@@ -86,7 +89,7 @@
    * Runs all effects of a given type for the given set of property changes
    * on an instance.
    *
-   * @param {Object} inst The instance with effects to run
+   * @param {PropertyEffectsInstance | Polymer_PropertyEffects} inst The instance with effects to run
```

Comment:
What does this mean?  Seems like one or the other, no?  Why doesn't it need to be a union of those two things?